pipeline/scripts/inference_pipeline.jl

```diff
@@ -1,38 +1,46 @@
 using DrWatson
 @quickactivate "Analysis pipeline"
 
+# Include the AnalysisPipeline module
 include(srcdir("AnalysisPipeline.jl"))
+# Include the common parameter values
 include(scriptsdir("common_param_values.jl"))
+# Include the common inference scenarios
+include(scriptsdir("common_scenarios.jl"))
```

Comment:
As above I think we need an issue to discuss other pipeline approaches that are not script based sourcing as this is probably the least stable/ good practice option of those we could choose imo